feature-foryou/src/main/res/values/strings.xml

```diff
@@ -24,8 +24,8 @@
     <string name="onboarding_guidance_title">What are you interested in?</string>
     <string name="onboarding_guidance_subtitle">Updates from topics you follow will appear here. Follow some things to get started.</string>
     <string name="top_app_bar_title">Now in Android</string>
-    <string name="top_app_bar_action_button_content_desc">My account</string>
-    <string name="top_app_bar_navigation_button_content_desc">Search</string>
+    <string name="for_you_top_app_bar_action_button_content_desc">My account</string>
```

Comment:
What do you think about renaming those to their actual values? Such as `for_you_top_app_bar_action_account` and `for_you_top_app_bar_action_search`

Reply:
i like it, done